axes/conf.py

```diff
@@ -53,3 +53,11 @@ class MyAppConf(AppConf):
 
     # message to show when locked out and have cooloff disabled
     PERMALOCK_MESSAGE = 'Account locked: too many login attempts. Contact an admin to unlock your account.'
+
+    # set to the amount of proxies you are using if you have reverse proxies configured
+    IPWARE_PROXY_COUNT = None
+
+    # set to a trusted header value if your reverse proxy is configured to set it to the HTTP request
+    IPWARE_REQUEST_HEADER_ORDER = (
+        'REMOTE_ADDR',
+    )
```

Comment:
It might be better to give advanced users ability to overwrite, and fallback on defaults for the rest.

```python
from django.conf import settings

AXES_META_PRECEDENCE_ORDER = getattr(settings,
    'AXES_META_PRECEDENCE_ORDER', getattr(settings,
        'IPWARE_META_PRECEDENCE_ORDER', (
             'REMOTE_ADDR',
              # ...
        )
     )
  )
}

# AXES_META_PRECEDENCE_ORDER is used so people who don't care about ipware don't have to know about it.
# IPWARE_META_PRECEDENCE_ORDER is used so people who already use ipware don't have to worry about it.
# Ipware can take all the params via kwargs, and it can be use directly. No need to worry about mixed settings.
```

Reply:
Without something like this, our application will be broken by this change. Our application already has a fix in place by setting `IPWARE_META_PRECEDENCE_ORDER=('HTTP_X_FORWARDED_FOR',),`.